app/controllers/foreman_inventory_upload/uploads_controller.rb

```diff
@@ -27,5 +27,13 @@ def throw_flash_error(message)
         :error_msg => message
       )
     end
+
+    def toggle_auto_upload
```

Comment:
I think it would be better to stick with REST notion:

``` ruby
def auto_upload(value)
  Setting[:allow_auto_inventory_upload] = value
end
```

This way we are not dependent on UI implementation